app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/settings/SettingsDownloadsLimitScreen.kt

```diff
@@ -73,7 +84,11 @@ enum class DownloadsLimit(
 private fun SettingsThemeScreenPreview() {
     SwissTransferTheme {
         Surface {
-            SettingsDownloadsLimitScreen {}
+            SettingsDownloadsLimitScreen(
+                downloadLimit = DownloadLimit.TWOHUNDREDFIFTY,
```

Comment:
I can't comment at the correct location, so I'll put my comment here :

In the `enum class DownloadsLimitOption`, rename `TWOHUNDREDFIFTY` into `TWO_HUNDRED_FIFTY` and `ONEHUNDRED` into `ONE_HUNDRED`.

Reply:
Fixed in another PR.